client/src/components/base/Modal/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+type ModalProps = {
+  title: React.ReactNode;
+  children: React.ReactNode;
+  isOpen: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+};
+
+const Modal = ({ title, children, isOpen, onClose, onConfirm }: ModalProps) => {
```

Comment:
```suggestion
import { FC } from 'react';
const Modal:FC<ModalProps> = ({ title, children, isOpen, onClose, onConfirm }) => {
```

이렇게 props를 정의할 수 도 있습니다. (참고하세요~)

Reply:
React.FC가 버그가 있어서 사용하지 않는 사람들이 있다고 합니다.
https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/
어떤게 좋을까요?

Reply:
읽어보니까 defaultProps를 사용할 때 에러가 발생한다는 거 같은데 
defaultProps를 사용할 일이 있으면 사용하면 안 되겠네요!

저는 defaultProps를 사용하지 않고 매개변수 선언부에 `{ name = '' }` 이런 식으로 사용해 와서 저 에러를 못 봤던거 같네요.

편의에 따라 달리 사용하면 될 거 같아요~!

Reply:
고런 문제라면 `FC` 대신 `VFC`도 추천드립니다~~